coretex/_task/worker/utils.py

```diff
@@ -43,7 +43,10 @@ def initializeLogger(taskRunId: int) -> None:
         style = "%",
     )
 
-    workerLogPath = folder_manager.logs / f"task_run_worker_{taskRunId}.log"
+    taskRunLogs = folder_manager.runsLogDirectory / str(taskRunId)
+    taskRunLogs.mkdir(parents = True, exist_ok = True)
```

Comment:
Let's create a function in `folder_manager` called `getRunLogsDir` which will have `taskRunId` as a parameter and return a directory for storing logs of that Task Run.